lib/server/__mocks__/tiny-lr.js

```diff
@@ -5,6 +5,8 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-module.exports = {
-  LIVE_RELOAD_PORT: 35729,
+const tinylrServer = {
+  listen: jest.fn(),
```

Comment:
This file has no relationship to the `constants` file, right? You just changed the `constants` file and then made a file move. I am just wondering why GitHub thinks this is a file move.

Reply:
That's right, the two files are unrelated. I actually deleted the `constants` file in a separate commit, and added the `tiny-lr` mock in a later one, so I'm not sure why GitHub thinks this was a file move 😕